config/routes.rb

```diff
@@ -1,22 +1,14 @@
 Webapp::Application.routes.draw do
 
-  resources :home do
+  resources :assignments, only: [] do
```

Comment:
`resources :assignments, only: []` won't add any of the standard RESTful routes.  The assignments controller doesn't implement any of them, they just represent a security risk.